docs/verify/index.mdx

```diff
@@ -17,7 +17,17 @@ Stabilizer circuits, also known as Clifford circuits, are a useful tool for acco
 
 For general quantum circuits, the following tools are available to verify your quantum programs:
 
-- For simulating circuits when using Qiskit Runtime, use its [local testing mode.](local-testing-mode) 
+- For simulating circuits when using Qiskit Runtime, use its [local testing mode.](local-testing-mode)
 - For exact simulation of small quantum circuits, you can use the reference primitives included with Qiskit. See [Exact simulation with Qiskit primitives](simulate-with-qiskit-primitives).
 - For higher-performance simulation that can handle larger circuits, or to incorporate noise models into your simulation, use [Qiskit Aer](https://qiskit.org/ecosystem/aer/), a project that is part of the [Qiskit Ecosystem](https://qiskit.github.io/ecosystem/). See [Exact and noisy simulation with Qiskit Aer primitives](simulate-with-qiskit-aer) (V1 primitive support only). For V2 primitive support, use the [Qiskit Runtime local testing mode](local-testing-mode) with a Qiskit Aer backend.
-- To build custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
\ No newline at end of file
+- To build custom noise models, use the [`noise`](https://qiskit.org/ecosystem/aer/apidocs/aer_noise.html) module of Qiskit Aer. See [Building noise models](building_noise_models).
+
+## Hardware considerations
+
+Several factors impact how much memory quantum simulation requires, so there are no exact hardware requirements for simulation, but there are some guidelines you can follow.
+
+- The only requirement to run Qiskit is a functioning Python environment.  For details, see the [Operating system support](../start/install#operating-system-support) section.
+- Because the requirements for simulating quantum circuits scale exponentially with the number of qubits, the available hardware limits the number of qubits that can be simulated. For example, a system with 4GB RAM can simulate approximately 27 qubits.
+- More or less available memory will not result in better or worse results (assuming results are returned).
```

Comment:
This might be more clear?

```suggestion
- More or less available memory will not result in better or worse quality of results (assuming results are returned).
```

Reply:
Haha that's what it was originally but I suggested removing the word "quality" in my review. I think we should stick with simplicity unless it feels very unclear, then maybe a reword would help

Reply:
I personally find it useful because it clarified what we mean by "better". If you have stronger hardware, you will get _faster_ results; that is better in some sense. But the quality/fidelity/accurateness is not impacted by your hardware.

Reply:
If it's clear what "quality" means in this sense, then sure. It feels redundant in many circumstances to say something is of "better quality" unless quality is defined. Can we say "more accurate" or "improved fidelity" or something like this?

Reply:
'll try to reword for top notch quality of clarity :)